indico_jacow/plugin.py

```diff
@@ -39,22 +46,70 @@ class JACOWPlugin(IndicoPlugin):
 
     def init(self):
         super().init()
-        self.template_hook('abstract-list-options', self.inject_abstract_export_button)
-        self.template_hook('contribution-list-options', self.inject_contribution_export_button)
-        self.connect(signals.event.sidemenu, self.extend_event_menu)
-        self.connect(signals.menu.items, self.add_sidemenu_item, sender='event-management-sidemenu')
-
-    def inject_abstract_export_button(self, event=None):
+        self.template_hook('abstract-list-options', self._inject_abstract_export_button)
+        self.template_hook('contribution-list-options', self._inject_contribution_export_button)
+        self.template_hook('custom-affiliation', self._inject_custom_affiliation)
+        self.connect(signals.core.form_validated, self._form_validated)
+        self.connect(signals.event.abstract_accepted, self._abstract_accepted)
+        self.connect(signals.event.hide_affiliation_field, self._hide_affiliation_field)
+        self.connect(signals.event.sidemenu, self._extend_event_menu)
+        self.connect(signals.menu.items, self._add_sidemenu_item, sender='event-management-sidemenu')
+        self.connect(signals.plugin.schema_pre_load, self._person_link_schema_pre_load, sender=PersonLinkSchema)
+        self.connect(signals.plugin.schema_post_dump, self._person_link_schema_post_dump, sender=PersonLinkSchema)
+        self.inject_bundle('main.js', WPContributions)
+        self.inject_bundle('main.js', WPDisplayAbstracts)
+        self.inject_bundle('main.js', WPManageAbstracts)
+        self.inject_bundle('main.js', WPManageContributions)
+        self.inject_bundle('main.js', WPMyContributions)
+        self.inject_bundle('main.css', WPContributions)
+        self.inject_bundle('main.css', WPDisplayAbstracts)
+        self.inject_bundle('main.css', WPManageAbstracts)
+        self.inject_bundle('main.css', WPManageContributions)
+        self.inject_bundle('main.css', WPMyContributions)
```

Comment:
```suggestion
        wps = (WPContributions, WPDisplayAbstracts, WPManageAbstracts, WPManageContributions, WPMyContributions)
        self.inject_bundle('main.js', wps)
        self.inject_bundle('main.css', wps)
```

Not tested though, but it should work since by default `inject_bundle` uses `issubclass()` to check whether to inject the bundle, and that accepts a tuple (just like `isinstance` does).